client/src/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import '../index';
+import {BoldBI} from '@boldbi/boldbi-embedded-sdk';
+
+
+//NodeJs application would be run on http://localhost:8080, which needs to be set as `apiHost`
+const apiHost="http://localhost:8080";
+
+//Url of the authorizationserver action in ValuesController of the NodeJs application
```

Comment:
//Url of the authorizationserver action in the ValuesController of the NodeJs application.

Reply:
updated this line